docs/detailed_test_config.rst

```diff
@@ -232,13 +232,16 @@ to explore the system as is. Adding a top-level::
 
     interactive-on-error: true
 
-as a config file for teuthology will make that possible. With that
-option, any *task* that fails, will have the ``interactive`` task
-called after it. This means that before any cleanup happens, you get a
-chance to inspect the system -- both through Teuthology and via extra
-SSH connections -- and the cleanup completes only when you choose so.
+as a config file for teuthology or passing ``--interactive-on-error`` flag will
+make that possible. With this option, any *task* that fails, will have the
+``interactive`` task called after it.  This means that before any cleanup
+happens, you get a chance to inspect the system -- both through Teuthology and
+via extra SSH connections -- and the cleanup completes only when you choose so.
 Just exit the interactive Python session to continue the cleanup.
```

Comment:
I would not change this paragraph at all. Maybe just add a single sentence to the end:
The interactive-on-error can be enabled with `--interactive-on-error` option with ``teuthology`` command.

Reply:
updated

Reply:
extra comma after "fails".  trailing "so" can be removed.

Reply:
@dmick updated